src/components/text-input/stories/framework-react.tsx

```diff
@@ -16,7 +17,19 @@ export const ReactDaikinTextInput = createComponent({
 });
 
 export const metadata: Meta<DaikinTextInputStoryArgs> = {
-  component: ({ ...props }: DaikinTextInputStoryArgs) => (
-    <ReactDaikinTextInput {...props} />
-  ),
+  component: ({ ...props }: DaikinTextInputStoryArgs) => {
+    const additionalClassNames = {
+      "": "",
+      resizeLarge:
+        "[&>daikin-text-input]:w-[800px] [&>daikin-text-input]:h-[320px]",
+      resizeSmall:
+        "[&>daikin-text-input]:w-[160px] [&>daikin-text-input]:h-[40px]",
+    }[props.vrtArgs ?? ""];
+
+    return (
+      <div className={additionalClassNames}>
+        <ReactDaikinTextInput {...props} />
+      </div>
+    );
```

Comment:
This might be much straightforward.

```suggestion
    const additionalClassNames = {
      "": "",
      resizeLarge:
        "w-[800px] h-[320px]",
      resizeSmall:
        "w-[160px] h-[40px]",
    }[props.vrtArgs ?? ""];

    return (
      <ReactDaikinTextInput className={additionalClassNames} {...props} />
    );
```

Reply:
@yodas7 
I'm sorry. I tried the above code with the React version of Storybook, but it didn't work.

Reply:
@poetrainy 
I tried it out and it worked.
I pushed the changes, so please check them.